samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -56,12 +56,14 @@ static struct broadcast_sink_stream {
 	size_t loss_cnt;
 	size_t error_cnt;
 	size_t valid_cnt;
+	size_t in_buf_len;
+	struct net_buf *in_buf;
```

Comment:
```suggestion
	struct net_buf *in_buf;
```

The length is part of the `net_buf` structure